packages/blockfrost/test/e2e/queryTransactions.test.ts

```diff
@@ -26,5 +26,115 @@ describe('blockfrostWalletProvider', () => {
         txId: '2db6592c4782064295295b365c2e8ce84084fa24b1b3f5834f3c6b65268b7878'
       });
     });
+    // TODO: waiting for fix in db-sync/blockfrost
```

Comment:
Is there a link we can reference?

Reply:
I think it's https://github.com/input-output-hk/cardano-db-sync/issues/1019, but blockfrost need to update to include it